src/ClientResolver.php

```diff
@@ -917,17 +926,41 @@ public static function _apply_http_handler($value, array &$args, HandlerList $li
         );
     }
 
+    public static function _apply_app_id($inputAppId, array &$args)
```

Comment:
let's stick with the convention we use in this class of calling the value `$value`